### PR TITLE
fix(dashboard): fixed the missing sort parameter

### PR DESCRIPTION
**Pull request by Fewwy**
Fixed the missing parameter in the URL for the dashboard.

### CODE REVIEW

File: src/SmartComponents/Advisor/Advisor.js [hunk 1]
```diff
@@ -56,7 +56,7 @@ const Advisor = () => {
 
     const urlRest = `&reports_shown=true&impacting=true&offset=0&limit=10${selectedTags?.length ?
         `&tags=${selectedTags?.join()}` : ''}${workloads?.SAP ? '&sap_system=true' : ''}${SID?.length ? `&sap_sids=${SID?.join()}` : ''}`;
-    const totalRiskUrl = risk => `${UI_BASE}/advisor/recommendations?total_risk=${risk}${urlRest}`;
+    const totalRiskUrl = risk => `${UI_BASE}/advisor/recommendations?sort=total_risk=${risk}${urlRest}`;
```
Comment (gkarat) [inline src/SmartComponents/Advisor/Advisor.js:59]: Perhaps you meant
```suggestion
    const totalRiskUrl = risk => `${UI_BASE}/advisor/recommendations?sort=total_risk&total_risk=${risk}${urlRest}`;
```
?

File: package.json [hunk 2]
```diff
@@ -106,7 +106,7 @@
     "babel-plugin-transform-imports": "2.0.0",
     "babel-polyfill": "6.26.0",
     "css-loader": "6.2.0",
-    "cypress": "^12.1.0",
+    "cypress": "^11.1.0",
```
Comment (gkarat) [inline package.json:109]: Why it's needed here?

File: package.json [hunk 3]
```diff
@@ -67,10 +67,10 @@
     "@patternfly/react-icons": "^4.72.3",
     "@patternfly/react-table": "^4.90.3",
     "@patternfly/react-tokens": "^4.73.3",
-    "@redhat-cloud-services/frontend-components": "^3.9.2",
+    "@redhat-cloud-services/frontend-components": "^3.9.28",
     "@redhat-cloud-services/frontend-components-notifications": "^3.2.7",
     "@redhat-cloud-services/frontend-components-translations": "3.2.3",
-    "@redhat-cloud-services/frontend-components-utilities": "3.2.3",
+    "@redhat-cloud-services/frontend-components-utilities": "3.3.11",
```
Comment (gkarat) [inline package.json:73]: Why it's needed here?

**Review by gkarat — CHANGES_REQUESTED** [review]

File: src/SmartComponents/Advisor/Advisor.js (hunk 1 above)
Comment (gkarat) [reply src/SmartComponents/Advisor/Advisor.js:59]: And the sort should be descending, thus you have to use `-total_risk`

File: package.json (hunk 3 above)
Comment (Fewwy) [reply package.json:73]: In the description

File: src/SmartComponents/Advisor/Advisor.js (hunk 1 above)
Comment (Fewwy) [reply src/SmartComponents/Advisor/Advisor.js]: True, updated the PR

File: src/SmartComponents/Advisor/Advisor.js [hunk 4]
```diff
@@ -56,7 +56,7 @@ const Advisor = () => {
 
     const urlRest = `&reports_shown=true&impacting=true&offset=0&limit=10${selectedTags?.length ?
         `&tags=${selectedTags?.join()}` : ''}${workloads?.SAP ? '&sap_system=true' : ''}${SID?.length ? `&sap_sids=${SID?.join()}` : ''}`;
-    const totalRiskUrl = risk => `${UI_BASE}/advisor/recommendations?total_risk=${risk}${urlRest}`;
+    const totalRiskUrl = risk => `${UI_BASE}/advisor/recommendations?sort=total_risk&total_risk=${risk}${urlRest}`;
```
Comment (gkarat) [inline src/SmartComponents/Advisor/Advisor.js:59]:
```suggestion
    const totalRiskUrl = risk => `${UI_BASE}/advisor/recommendations?sort=-total_risk&total_risk=${risk}${urlRest}`;
```
must be descending

Comment (Fewwy) [reply src/SmartComponents/Advisor/Advisor.js]: Yep, done

**Review by adonispuente — APPROVED** [review]
LGTM, thank you for making the quick changes !

**Pull request merged**